### ISSUE TITLE
A .tsrx file OXC cannot parse aborts the whole lint batch without naming the file

### ISSUE THREAD

**Issue by thejackshelton** [tsrx-org/oxc#79]
Found while wiring Markless onto 0.11.0. Its TypeScript-plugin test fixtures include editor-completion probes with deliberately incomplete expressions, for example:

```tsrx
export function TypeScriptCapabilities() @{
	const element = document.createElement('div');
	document./*M3_DOM*/;
	<div>{element.dataset}</div>
}
```

`oxlint .` over the repository then exits 2 with only:

```
oxlint (oxc-tsrx): OXC parse failed: Unexpected token
```

Two problems:

1. **The whole batch aborts.** 1,947 files had already been linted clean; one file the projected TSX cannot be parsed from takes the run down, so nothing is reported.
2. **The file is not named.** The error carries no path, so the user has to bisect. Compare `broken.tsrx` with an unterminated JSX element in the same repo: the TSRX scanner turns that into a positioned per-file diagnostic and the batch continues (`tests/config/native-lint-config.test.mjs`: "a .tsrx syntax error is a positioned diagnostic that leaves the rest of the batch reporting").

Expected: an OXC parse failure on the projected copy becomes a per-file `error` diagnostic with the file name and, where possible, a position mapped back to the authored source, and the rest of the batch still reports. `oxfmt --check` already names the file in this case (`…/typescript.tsrx: OXC formatter parse failed: Unexpected token`), so the lint lane should at least match that.

Origin: `crates/oxc_adapter/src/toolchain/session.rs` (`OXC parse failed: {detail}`) surfaces as a session error rather than a file result. Workaround for consumers: add such fixture directories to `ignorePatterns`.

**Comment by thejackshelton** [comment]
Released in **0.12.0** (#81). A file OXC cannot parse is now one named, positioned `error` per parser diagnostic in the report, the batch continues, and the exit code is the ordinary 1 for errors rather than the tool-failure 2. The language server positions the same error in the buffer.
